c-low-level/Dockerfile

```diff
@@ -1,36 +1,31 @@
 # syntax=docker.io/docker/dockerfile:1
+ARG MACHINE_EMULATOR_TOOLS_VERSION=0.16.1
 FROM --platform=linux/riscv64 ubuntu:22.04 AS builder
 
 ARG DEBIAN_FRONTEND=noninteractive
 RUN <<EOF
 set -e
 apt-get update
 apt-get install -y --no-install-recommends \
-  autoconf \
-  automake \
   build-essential \
-  ca-certificates \
-  curl \
   libtool \
-  wget
+  pkg-config
 rm -rf /var/lib/apt/lists/*
 EOF
 
-COPY --from=cartesi/sdk:0.9.0 /include/linux/cartesi /include/linux/cartesi
+ARG MACHINE_EMULATOR_TOOLS_VERSION
+ADD --checksum=sha256:ac93f0b8b2c8be85e1b3956f3e0206009fbb8c73f9327948a811b1864a2cee8d \
+  https://github.com/cartesi/machine-emulator-tools/releases/download/v${MACHINE_EMULATOR_TOOLS_VERSION}/libcmt-dev-v${MACHINE_EMULATOR_TOOLS_VERSION}.deb \
+  /tmp/libcmt-dev-v${MACHINE_EMULATOR_TOOLS_VERSION}.deb
+RUN dpkg -i /tmp/libcmt-dev-v${MACHINE_EMULATOR_TOOLS_VERSION}.deb \
+  && rm /tmp/libcmt-dev-v${MACHINE_EMULATOR_TOOLS_VERSION}.deb
+
 WORKDIR /opt/cartesi/dapp
-COPY . .
```

Comment:
can we just `COPY . .` here? Because then it doesn't need to change when the dev adds more `.c` or `.h` files to his dapp

Reply:
done